src/retyped/flowprinter.re

```diff
@@ -32,9 +29,14 @@ let rec show_type =
                 | _ => name ^ ": " ^ show_type type_of
                 }
             )
-            params
-        ) ^
-      "): " ^ show_type return
+            params;
```

Comment:
I'm not sure what the flow printer is used for, but I thought I'd at least try to expand it to print variadics as such.